tests/unit/package_managers/test_gomod.py

```diff
@@ -666,18 +666,6 @@ def test_go_list_cmd_failure(
 def test_deduplicate_resolved_modules():
     # as reported by "go list -deps all"
     package_modules = [
-        # local replacement
-        ParsedModule(
-            path="github.com/my-org/local-replacement",
-            version="v1.0.0",
-            replace=ParsedModule(path="./local-folder"),
-        ),
-        # dependency replacement
-        ParsedModule(
-            path="github.com/my-org/my-dep",
-            version="v2.0.0",
-            replace=ParsedModule(path="github.com/another-org/another-dep", version="v2.0.1"),
-        ),
```

Comment:
I think we should keep these cases

We still support replacements defined in the source code (`replace github.com/my-org/my-dep v2.0.0 => github.com/my-org/my-dep v2.0.1` in go.mod). We're only dropping support for dependency replacements specified dynamically on the command line.

This unit test also applies to source-code replacements, not only the CLI ones